tests/runtests.bats

```diff
@@ -77,6 +80,21 @@ cleanup() {
   cleanup
 }
 
+@test "bring up with two disks" {
+  export VAGRANT_CWD=tests/two_disks
+  cleanup
+  run create_box_with_two_disks
```

Comment:
It seems like you've already got the code being called here for creating the box with two disks, so can the other entries be removed?

Reply:
Yes i can remove it.